storybook/src/components/Alert/Alert.test.stories.tsx

```diff
@@ -20,6 +21,10 @@ export default meta
 type Story = StoryObj<typeof meta>
 
 export const Test: Story = {
+  args: {
+    closeable: true,
+    onClose: fn(),
+  },
```

Comment:
I don't think we need these `args` anymore, right? Or do you want to show the close button for most examples?